homeassistant/components/raspberrypi/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "raspberrypi",
```

Comment:
Should we add a new `integration_type` that is "hardware" ?

Reply:
What would the difference be between an `integration` integration and a `hardware` integration?

Reply:
It's used in frontend to make sure they are not shown to users.

Reply:
OK, makes sense 👍